validation/validation_test.go

```diff
@@ -114,9 +114,9 @@ func TestValidateCreate(t *testing.T) {
 		errExpected bool
 		err         error
 	}{
-		{"", false, rerr.NewError(rerr.CreateWithID, "'Create' MUST generate an error if the ID is not provided", rspecs.Version)},
-		{containerID, true, rerr.NewError(rerr.CreateNewContainer, "'Create' MUST create a new container", rspecs.Version)},
-		{containerID, false, rerr.NewError(rerr.CreateWithUniqueID, "'Create' MUST generate an error if the ID provided is not unique", rspecs.Version)},
+		{"", false, specerror.NewError(specerror.CreateWithID, "'Create' MUST generate an error if the ID is not provided", rspecs.Version)},
```

Comment:
I'm fine with new format of NewError(), but it seems you leave behind here three place. I wonder why there is no error when compiling.

Reply:
validation_test.go will not run in our CI.

Reply:
> I'm fine with new format of NewError(), but it seems you leave behind here three place.

Oops.  Thanks.  Fixed with c3a8d2c → 7e95c6f.